app/controllers/chits.rb

```diff
@@ -0,0 +1,27 @@
+class Chitter < Sinatra::Base
```

Comment:
I'd expect my controllers to be classes in their own right -> we'll get to this when we look at Rails.